objects/official/ride/openrct2.ride.alpine_coaster/object.json

```diff
@@ -60,23 +40,16 @@
                     "flatToGentleSlopeBankedTransitions": true,
                     "diagonalGentleSlopeBankedTransitions": true
                 },
-                "loadingPositions": [
-                    -1,
-                    1
-                ]
+                "loadingPositions": [-1, 1]
             }
         ]
     },
     "strings": {
-        "name": {
-            "en-GB": "Alpine Coaster Cars"
-        },
+        "name": { "en-GB": "Alpine Coaster Cars" },
```

Comment:
This is going to suck when inserting languages.

Reply:
The languages are synchronized with a new script, the end user doesn't have to deal with this. Also JSON is not made to be human friendly at all, using JSON was the wrong choice so for consistency reasons we decided to format all the objects first with the new script to minimize diffs when synchronized.

Reply:
I fixed this.